images/single-cloud/oci/sles-sap-byos/content.yaml

```diff
@@ -0,0 +1,30 @@
+image:
+  preferences:
+    - _include:
+        - base/common
+        - csp/oci/settings/sap/byos
+  packages:
+    - _attributes:
+        type: bootstrap
+      _include: base/bootstrap
+    - _attributes:
+        type: image
+      _include:
+        - base/common
+        - base/sle
+        - base/byos
```

Comment:
There's no `base/byos`, should be removed.

Reply:
Ditto for the other two uses?

Reply:
Yep.